python/ml4ir/applications/classification/model/classification_model.py

```diff
@@ -163,12 +187,12 @@ def calculate_metric_on_batch(self, metric, predictions, batch_size, group_name=
                 "size": predictions.shape[0]}
 
     def predict(
-        self,
-        test_dataset: data.TFRecordDataset,
-        inference_signature: str = "serving_default",
-        additional_features: dict = {},
-        logs_dir: Optional[str] = None,
-        logging_frequency: int = 25,
+            self,
```

Comment:
Pycharm is auto formatting this for me. Do you guys have different configuration by any chance for auto-formatting?